code/modules/mob/living/simple_animal/hostile/mining_fauna/outsiders.dm

```diff
@@ -0,0 +1,81 @@
+//contains:
+//~shadow
+//ancient one
+
+////////////////////////////////////////
+///////////////Shadowmancer/////////////
+////////////////////////////////////////
+
+/obj/item/projectile/energy/shadow
+	name = "heavenly orb"
+	icon_state = "magicm"
+	check_armour = "energy"
+	damage = 15
+	damage_type = BURN
+	pass_flags = PASSTABLE | PASSGLASS | PASSGRILLE
+	embed = 0
+	incinerate = 5
+
+/obj/item/projectile/energy/shadow/Initialize()
+	..()
+	set_light(1,2,LIGHT_COLOR_FIRE)
```

Comment:
Might as well put these as vars on the mob instead of a `set_light` call in `Initialize()`.
```
light_range = 1
light_power = 2
light_color = LIGHT_COLOR_FIRE
```

The range of 1 is also going to be clamped to 1.4, the minimum for this var.